docs/translations.md

```diff
@@ -1,43 +1,672 @@
+
+Relay staff can check translation status on the Pontoon project sites. They may use
+the completeness of translated strings to decide to turn on a feature for a region.
+
+[amo-relay]: https://addons.mozilla.org/en-US/firefox/addon/private-relay/
+[pontoon]: https://pontoon.mozilla.org/
+[pt-ca-valencia]: https://pontoon.mozilla.org/ca-valencia/
+[pt-de]: https://pontoon.mozilla.org/de/
+[pt-es-ES]: https://pontoon.mozilla.org/es-ES/
+[pt-fr]: https://pontoon.mozilla.org/fr/
+[pt-ia-relay]: https://pontoon.mozilla.org/ia/firefox-relay-website/
+[pt-lb]: https://pontoon.mozilla.org/lb/
+[pt-proj-addon]: https://pontoon.mozilla.org/projects/firefox-relay-add-on/
+[pt-proj-amo]: https://pontoon.mozilla.org/projects/amo-frontend/
+[pt-proj-bedrock]: https://pontoon.mozilla.org/projects/mozillaorg/
+[pt-proj-fxa]: https://pontoon.mozilla.org/projects/firefox-accounts/
+[pt-proj-relay]: https://pontoon.mozilla.org/projects/firefox-relay-website/
+[pt-proj-sumo]: https://pontoon.mozilla.org/projects/sumo/
+[pt-projs]: https://pontoon.mozilla.org/projects/
+[pt-scn]: https://pontoon.mozilla.org/scn/
+[pt-teams]: https://pontoon.mozilla.org/teams/
+[sumo-relay]: https://support.mozilla.org/en-US/products/relay
+
+### Shipping Translations
+
+Pontoon syncs translations between its database and code repositories, most hosted in
+the [Mozilla Localization Team][mozilla-l10n] GitHub organization. The Pontoon project
+page has a link to the project repository. Some relevant repositories include:
+
+- [mozilla-l10n/fx-private-relay-i10n][] for the Firefox Relay Website
+- [mozilla-l10n/fx-private-relay-add-on-l10n][] for the Firefox Relay Add-on
+- [mozilla/fxa-content-server-l10n][] for Firefox Accounts, including subscription pages
+- [mozilla-l10n/www-l10n][] for mozilla.org
+- [mozilla-l10n/sumo-l10n][] for support.mozilla.org
+- [mozilla/addons-frontend][] for addons.mozilla.org
+
+Relay projects include the translations as [git submodules][git-submodules], such as
+[privaterelay/locales][]. A submodule references a specific commit in a separate repository.
+A [GitHub action][ga-l10n-sync] updates this commit reference nightly to get the
+latest translations from Pontoon. This strategy works better for Relay than other
+strategies, such as letting Pontoon write directly to the code repository, or
+periodically syncing translations.
+
+Translations for Relay, and many projects at Mozilla, use the [Fluent format][fluent].
+The ["en" bundle][relay-l10n-en] (American / generic English) has the source
+translations, split into functional areas, such as website pages. The other language
+bundles are translations of the `"en"` bundle, coordinated in Pontoon. The continuous
+integration process creates Docker images that include the Fluent files, both in the
+source form used by the [Django backend][ftl-bundles], and embedded into the website
+JavaScript.
+
+Relay developers create new translatable strings when updating the service. The website
```

Comment:
I need to decide if it will be spelled "front-end", "frontend", or "front end". I think "front end" is the correct spelling of the noun, "front-end" as adjective.